mysite/settings.py

```diff
@@ -1,23 +1,21 @@
 """Django settings for mysite project."""
 
-import os
-import dj_database_url
 
-ON_HEROKU = os.environ.get('ON_HEROKU')
+import environ
 
-# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+env = environ.Env(DEBUG=(bool, False),)
 
+BASE_DIR = environ.Path(__file__) - 2
+environ.Env.read_env()
 
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '_u)sp0-q^_c!s645c-g5=whl97qsm&wxnw6rajs(hhri93wi9s'
+SECRET_KEY = env('SECRET_KEY', default='_u)sp0-q^_c!s645c-g5=whl97qsm&wxnw6rajs(hhri93wi9s')
```

Comment:
it's not in example. it's better to not have a default (except for .env example), because one MUST create smth not tracked in git for their prod. it's a security issue otherwise.